File: plugin_loader/src/server_api/sc/protocol/responses/LeftGameEvent.java
package sc.protocol.responses;

import com.thoughtworks.xstream.annotations.XStreamAlias;
import com.thoughtworks.xstream.annotations.XStreamAsAttribute;

@XStreamAlias(value="left")
public class LeftGameEvent
{
	@XStreamAsAttribute
	private String	roomId;

	public LeftGameEvent(String id)
	{
		this.roomId = id;
	}

	public String getRoomId()
	{
		return this.roomId;
	}
}

File: sdk/src/server-api/sc/protocol/responses/LeftGameEvent.kt
package sc.protocol.responses

import com.thoughtworks.xstream.annotations.XStreamAlias
import com.thoughtworks.xstream.annotations.XStreamAsAttribute

/** Indicates that the receiving Player left or was kicked from the game. */
@XStreamAlias(value = "left")
data class LeftGameEvent(
        @XStreamAsAttribute
        val roomId: String
): ProtocolMessage
